nvOC_miner_update.sh

```diff
@@ -26,353 +136,643 @@ function restart-if-needed {
+  if [[ $CUDA_VER == "8.0" ]]
+  then
+    update-symlink ${NVOC_MINERS}/xmr_stak/${xmr_stak_ver_8} recommended
+  fi
   restart-if-needed
 else
-  echo "MSFTccminer is already up-to-date"
+  echo "xmr-stak is already up-to-date"
 fi
 
+echo "Checking xmr_stak ${xmr_stak_ver_9}"
+if [[ ! -d ${NVOC_MINERS}/xmr_stak/${xmr_stak_ver_9} ]]
+then
+  echo "Extracting xmr-stak"
+  mkdir -p ${NVOC_MINERS}/xmr_stak/${xmr_stak_ver_9}/
+  stop-if-needed "[x]mr-stak"
+  tar -xvJf ${NVOC_MINERS}/xmr_stak/${xmr_stak_tarball_ver_9} -C ${NVOC_MINERS}/xmr_stak/${xmr_stak_ver_9}/ --strip 1
+  chmod a+x ${NVOC_MINERS}/xmr_stak/${xmr_stak_ver_8}/xmr_stak_miner
+  if [[ $CUDA_VER == "9.2" ]]
+  then
+    update-symlink ${NVOC_MINERS}/xmr_stak/${xmr_stak_ver_9} recommended
+    update-symlink ${NVOC_MINERS}/xmr_stak/${xmr_stak_ver_9} latest
+  fi
+  restart-if-needed
+else
+  echo "xmr-stak is already up-to-date"
+fi
+echo
 echo
 
-echo "Checking SUPRminer 1.5"
-if [ ! $(cat ${NVOC_MINERS}/SUPRminer/version | grep 1.5) ]
+
+echo "Checking Z-ENEMY miner"
+if [[ ! -d ${NVOC_MINERS}/ZENEMYminer/${ZENEMYminer_ver_8} ]]
 then
-  echo "Extracting SUPRminer"
-  mkdir -p ${NVOC_MINERS}/SUPRminer/
-  stop-if-needed "[S]UPRminer"
-  cat ${NVOC_MINERS}/SUPRminer/SUPRminer-1.5.tar.xz | tar -xJC ${NVOC_MINERS}/SUPRminer/ --strip 1
-  chmod a+x ${NVOC_MINERS}/SUPRminer/ccminer
+  echo "Extracting Z-ENEMY miner ${ZENEMYminer_ver_8} and making changes for CUDA-8"
+  mkdir -p ${NVOC_MINERS}/ZENEMYminer/${ZENEMYminer_ver_8}/
+  tar -xvJf ${NVOC_MINERS}/ZENEMYminer/${ZENEMYminer_tarball_ver_8} -C ${NVOC_MINERS}/ZENEMYminer/${ZENEMYminer_ver_8}/ --strip 1
+  chmod a+x  ${NVOC_MINERS}/ZENEMYminer/${ZENEMYminer_ver_8}/ccminer
+  stop-if-needed "[Z]ENEMYminer"
+  echo "Z-ENEMY miner for CUDA-8 updated"
+  echo "Use ${ZENEMYminer_ver_8} or recommended for ZENEMYminer_VERSION in 1bash"
+  if [[ $CUDA_VER == "8.0" ]]
+  then
+    update-symlink ${NVOC_MINERS}/ZENEMYminer/${ZENEMYminer_ver_8} recommended
+  fi
   restart-if-needed
 else
-  echo "SUPRminer is already up-to-date"
+  echo "Z-ENEMY miner for CUDA-8 is already up-to-date"
+  echo "Use ZENEMYminer_VERSION ${ZENEMYminer_ver_8} or recommended in 1bash"
+fi
+
+if [[ ! -d ${NVOC_MINERS}/ZENEMYminer/${ZENEMYminer_ver_9} ]]
+then
+  echo "Extracting Z-ENEMY miner and making changes for CUDA-9.2"
+  mkdir -p ${NVOC_MINERS}/ZENEMYminer/${ZENEMYminer_ver_9}/
+  tar -xvJf ${NVOC_MINERS}/ZENEMYminer/${ZENEMYminer_tarball_ver_9} -C ${NVOC_MINERS}/ZENEMYminer/${ZENEMYminer_ver_9}/ --strip 1
+  chmod a+x  ${NVOC_MINERS}/ZENEMYminer/${ZENEMYminer_ver_9}/ccminer
+  stop-if-needed "[Z]ENEMYminer"
+  echo "Z-ENEMY miner for CUDA-9.2 updated"
+  echo "Use latest or recommended or ${ZENEMYminer_ver_9} for ZENEMYminer_VERSION in 1bash"
+  if [[ $CUDA_VER == "9.2" ]]
+  then
+    update-symlink ${NVOC_MINERS}/ZENEMYminer/${ZENEMYminer_ver_9} recommended
+    update-symlink ${NVOC_MINERS}/ZENEMYminer/${ZENEMYminer_ver_9} latest
```

Comment:
This line should be outside of the if branch, it is not the only case,

Reply:
I mean it should be like this:

if [[ $CUDA_VER == "9.2" ]] 
then 
    update-symlink ${NVOC_MINERS}/ZENEMYminer/${ZENEMYminer_ver_9} recommended 
fi
update-symlink ${NVOC_MINERS}/ZENEMYminer/${ZENEMYminer_ver_9} latest 

If cuda version matches this miner gets "recommended",in any case it should be linked to latest.

Reply:
Will make the chnages